src/Domain/SalesforceWriteProxyRepository.php

```diff
@@ -122,6 +122,13 @@ public function pushSalesforcePending(\DateTimeImmutable $now): int
             self::MAX_PER_BULK_PUSH,
         );
 
+        if ($proxiesToCreate !== []) {
+            $count = count($proxiesToCreate);
+            $this->logger->error(
+                "Found $count pending items to push to SF, suggests push via Symfony Messenger failed"
```

Comment:
I think this might be a bit noisier than we want – at busy times, it might be fairly normal for the regularly scheduled push to have some messages in this state.

Reply:
Shall we keep it like this for now and be ready to change it if it becomes noisy? I'd definitely like to have extra visibility of what this is doing immediately after this goes to prod for the first time. Maybe we can / should make it less noisy by making this only activate when the backlog gets longer, e.g. replace fiveMinutesAgo with 20 minutes ago or an hour ago.

Reply:
Yep, thought about it a bit more and if a donation awaiting a push is this old I think there should probably be log noise from previous push attempts anyway. So one more message could help and is unlikely to hurt anything.